pennylane/templates/embeddings/amplitude.py

```diff
@@ -143,10 +143,19 @@ def __init__(self, features, wires, pad_with=None, normalize=False, pad=None, do
         features = self._preprocess(features, wires, pad_with, normalize)
         super().__init__(features, wires=wires, do_queue=do_queue)
 
+    def adjoint(self, do_queue=False):
+        return qml.templates.MottonenStatePreparation(self.parameters[0], wires=self.wires)
+
     def expand(self):
 
         with qml.tape.QuantumTape() as tape:
-            QubitStateVector(self.parameters[0], wires=self.wires)
+            if self.inverse:
+                self.adjoint()
+            else:
+                QubitStateVector(self.parameters[0], wires=self.wires)
+
+        if self.inverse:
+            tape.inv()
```

Comment:
Be sure to cover these lines. They're important and I'm not fully sure they behave the way you would expect.

Reply:
These lines are 'placeholders' to support the legacy `qml.inv()`. Actually, I'm surprised they are not covered, this test was designed to hit these lines:

https://github.com/PennyLaneAI/pennylane/blob/84c894367164e725917befcf3624aa0894742eb8/tests/transforms/test_adjoint.py#L115

Note that the test is parametrized over both `qml.adjoint()` and `qml.inv()`:

https://github.com/PennyLaneAI/pennylane/blob/84c894367164e725917befcf3624aa0894742eb8/tests/transforms/test_adjoint.py#L90

Reply:
The equivalent lines in the other embedding tests are being covered 🤔